apps/frontend/tailwind.config.ts

```diff
@@ -45,6 +45,8 @@ const config: Config = {
         quaternary: "var(--color-quaternary)",
         cinnamon: "var(--color-cinnamon)",
 
+        pink: "#ff9df8",
```

Comment:
Unideal but we can fix later

Reply:
geneerally using variabes is really good when we want to do things like dark mode, i'd try to find the right variable here